testcases/open_posix_testsuite/conformance/interfaces/aio_cancel/5-1.c

```diff
@@ -70,14 +72,12 @@ int main(void)
 	for (i = 0; i < BUF_NB; i++) {
 		aiocb[i] = calloc(1, sizeof(struct aiocb));
 		if (aiocb[i] == NULL) {
-			printf(TNAME " Error at malloc(): %s\n",
-			       strerror(errno));
+			printf(TNAME " Error at malloc(): %s\n", strerror(errno));
 			return PTS_UNRESOLVED;
 		}
```

Comment:
Can we instead allocate these buffers once in the main()? Or declare them statically?

It's mostly fine to allocate once and have it freed at program exit, but not so nice now that we call this function in a loop...

Reply:
You are right, I don't think there is any reason to do dynamic allocation at all. The size is known at compile time. I'll change it

Reply:
I replaced all dynamic memory allocations